### ISSUE TITLE
Release v1.44.1

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#3981]
:robot: A dedicated issue to run tests and deploy release [v1.44.1](https://github.com/deckhouse/deckhouse/milestone/162).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release related actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | azure | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref` is a tag `v1.44.1 | test-v1.44.1*` or a branch `main | release-1.44`
- `/e2e/run/<provider> git_ref_1 git_ref_2` will run e2e using provider and git_ref_1 and then switch to git_ref_2.
  - `provider` is one of `aws | azure | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref_1` is a release-* or main branch
  - `git_ref_2` is a release-* or main branch
- `/e2e/use/cri/<cri_name>` specifies which CRI to use for e2e test.
  - `cri_name` is one of `docker | containerd`
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.21 | 1.22 | 1.23 | 1.24 | 1.25 | 1.26`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag `v1.44.1 | test-v1.44.1*` or a branch `main | release-1.44`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.23'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws main
/e2e/use/cri/docker
/e2e/use/cri/containerd
/e2e/use/k8s/1.20
/e2e/use/k8s/1.23

This comment will run 4 e2e jobs on AWS with Docker and containerd
and with Kubernetes version 1.20 and 1.23 using image built from main branch.
```

```
/e2e/run/aws release-1.35 release-1.36
/e2e/use/cri/containerd
/e2e/use/k8s/1.23

This comment will create cluster in AWS using Deckhouse built from release-1.35 branch
and then switch to images built from release-1.36 branch.
'use' options are supported, cluster will use containerd and Kubernetes version 1.23.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build FE' job is finished.

**Note 3:**
No autobuild for release branch. Run this command after cherry-picking into release branch:
```
/build release-1.44
```

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [56748a](https://github.com/deckhouse/deckhouse/commit/56748a87f14d6a627bd715e5ec0e575565bc4fef) in branch **main**:
```
[testing] [ci] Execute module and matrix tests before build to unify local and GHA testing (#3740)

Signed-off-by: Gleb Maiorov <gleb.maiorov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [940687](https://github.com/deckhouse/deckhouse/commit/9406874735936bc1ddb1935b6d84cf1e95078176) in branch **main**:
```
[virtualization] Kubevirt v0.58.1 (#3989)

Signed-off-by: Andrei Kvapil <kvapss@gmail.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [e3d457](https://github.com/deckhouse/deckhouse/commit/e3d45705ef4ab958c8e9b79c21a61e5fea67a0cd) in branch **main**:
```
[docs][user-authz] Add links to the enableMultiTenancy module parameter (#3996)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [0a230d](https://github.com/deckhouse/deckhouse/commit/0a230df8e9f15973b23d8830278556fe08924432) in branch **main**:
```
[docs] Fix broken links (#3998)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [50dda7](https://github.com/deckhouse/deckhouse/commit/50dda7c8657f9e7d268892d90cd2039259eb267d) in branch **main**:
```
[user-authn] discover_dex_ca loops the list of secrets (#3842)

Signed-off-by: Konstantin Maltsev <konstantin.maltsev@flant.com>
Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
Co-authored-by: Konstantin Maltsev <konstantin.maltsev@flant.com>
Co-authored-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [3c72ba](https://github.com/deckhouse/deckhouse/commit/3c72bafffd08dbf2876b314764a6e626f381ec0b) in branch **main**:
```
[ingress-nginx] Refactor order_certificates hook for ingress-nginx module (#3950)

Signed-off-by: alexey.makhonin <alexey.makhonin@flant.com>
Co-authored-by: alexey.makhonin <alexey.makhonin@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [13382a](https://github.com/deckhouse/deckhouse/commit/13382a73fd1519d9bef1f5f5b67353f9e9927a9b) in branch **main**:
```
[docs] Fix sidebar menu (#4003)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [16d681](https://github.com/deckhouse/deckhouse/commit/16d6810487ac9969a2029b3dca9abb9ae48d1e93) in branch **main**:
```
[docs] Set DH_EDITION in lowercase (#3984)

Signed-off-by: Alexey Gavrilov <alexey.gavrilov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [ac87c1](https://github.com/deckhouse/deckhouse/commit/ac87c1222813d37673250b2ca1c0a64be4f830e6) in branch **main**:
```
Remove D8 prefix (#4004)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [fec5c5](https://github.com/deckhouse/deckhouse/commit/fec5c5db363d06997c20d8420e66afbaa44a9aa7) in branch **main**:
```
[ingress-nginx] Fix order certificates hook (#4015)

Signed-off-by: alexey.makhonin <alexey.makhonin@flant.com>
Co-authored-by: alexey.makhonin <alexey.makhonin@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [75f91e](https://github.com/deckhouse/deckhouse/commit/75f91e79a14dc5680959c820bac7fc25bb5ffab2) in branch **main**:
```
[cert-manager] fix linter (#4017)

Signed-off-by: alexey.makhonin <alexey.makhonin@flant.com>
Co-authored-by: alexey.makhonin <alexey.makhonin@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [e10b81](https://github.com/deckhouse/deckhouse/commit/e10b8170b1767b97b1eb5e0ebb4278e403067ae4) in branch **main**:
```
[deckhouse-config] disable webhook ha mode (#4014)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [bd7b3d](https://github.com/deckhouse/deckhouse/commit/bd7b3d74a697e3d4a3f98cdf1ccb717a49272587) in branch **main**:
```
Fix tolerations for virt-operator-strategy-dumper job (#3988)

Signed-off-by: Andrei Kvapil <kvapss@gmail.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [3e49bb](https://github.com/deckhouse/deckhouse/commit/3e49bbd18bf258649375bb43be6a55d7a27c3ebe) in branch **main**:
```
[log-shipper] Fix ratelimit alert (#4018)

Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [c71beb](https://github.com/deckhouse/deckhouse/commit/c71beb337ff7d868e00f7a5554e1b8ac06923f7b) in branch **main**:
```
Rename deprecated annotations for upmeter (#3986)

Signed-off-by: alexey.makhonin <alexey.makhonin@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [e7b315](https://github.com/deckhouse/deckhouse/commit/e7b315d1eafcc1af24a274498b3158df19af31a0) in branch **main**:
```
[docs] Yandex withNATInstance (#4025)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [8ec675](https://github.com/deckhouse/deckhouse/commit/8ec6754eb815a92b461c572d2d885a96afc0f01b) in branch **main**:
```
[docs] Minor updates (#4007)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [c138b0](https://github.com/deckhouse/deckhouse/commit/c138b0375cc5ef5a61202b7895b5806dc56889d4) in branch **main**:
```
[istio] D8IstioDeprecatedIstioVersionInstalled alert description fix (#4010)

Signed-off-by: Andrey Polovov <andrey.polovov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [5af9f0](https://github.com/deckhouse/deckhouse/commit/5af9f003d558b1f2bcbee7fd02895b0dbf3dba52) in branch **main**:
```
[cni-cilium] More docs (#4022)

* [cni-cilium] More docs

Signed-off-by: Andrey Klimentyev <andrey.klimentyev@flant.com>
Signed-off-by: Artem Kladov <6360800+z9r5@users.noreply.github.com>

---------

Signed-off-by: Andrey Klimentyev <andrey.klimentyev@flant.com>
Signed-off-by: Artem Kladov <6360800+z9r5@users.noreply.github.com>
Co-authored-by: Artem Kladov <6360800+z9r5@users.noreply.github.com>
```
